test/DynamoCoreTests/Nodes/ListTests.cs

```diff
@@ -2102,6 +2102,46 @@ public void TrueForAll_SimpleTest()
 
 		#endregion
 
+        #region Test AnyTrue
+
+        [Test]
+        public void AnyTrue_SimpleTest()
+        {
+            string openPath = Path.Combine(TestDirectory, @"core\list\AnyTrue_SimpleTest.dyn");
+            RunModel(openPath);
+
+            // check all the nodes and connectors are loaded
+            Assert.AreEqual(6, CurrentDynamoModel.CurrentWorkspace.Nodes.Count());
+            Assert.AreEqual(5, CurrentDynamoModel.CurrentWorkspace.Connectors.Count());
+            AssertPreviewValue("eb32799e0a26444081185a072e5c1c44", true);
+            AssertPreviewValue("dfc0430372ea40cb9b57c34551da69a6", false);
+            AssertPreviewValue("3391f6711c8d482eab224c881d512f73", true);
+            AssertPreviewValue("cfdc4008ae544881b80b0d2b807770e7", true);
+            AssertPreviewValue("2f20ee45e15149859b9e2d4548673079", true);
+		}
```

Comment:
Could you please take care of the formatting here? I think it may be an issue of using tabs vs spaces.

Reply:
Resolved in c38c6d8